src/ServiceMap.php

```diff
@@ -43,6 +43,11 @@ public function __construct(string $configPath)
         }
 
         foreach ($config['components'] ?? [] as $id => $component) {
+            if (!is_array($component)) {
```

Comment:
:thinking: maybe also `is_object`?

Reply:
I'm guessing `is_object` implies `!is_array` and that `is_object` is actually more reliable. 

I will update this and add a test-case for the edge-case where `!is_array`-behavior is not desired. :smile: